eng/vmr-sync.sh

```diff
@@ -1,63 +1,57 @@
-###
-### The whole process can be reproduced locally easily by running this script.
-### The patches are located in a folder in the 'dotnet/installer' repository.
-### At the moment of writing, the location is 'src/SourceBuild/patches' but to get
-### the up-to-date location, please see the 'patchesPath' property in
-### https://github.com/dotnet/dotnet/blob/main/src/source-mappings.json
-###
-### You will need to compare what is in the patch and what is in the file and fix the patch.
-###
-### The tooling that synchronizes the VMR will need to clone the various repositories.
-### It clones them into a temporary folder and re-uses them on future runs so it is advised
-### you dedicate a folder to this to speed up your re-runs.
-###
-### This script will synchronize the 'dotnet/dotnet' repo locally and let you inspect the changes.
+### The tooling that synchronizes the VMR will need to clone the various repositories into a temporary
+### folder. These clones can be re-used in future synchronizations so it is advised you dedicate a
+### folder to this to speed up your re-runs.
 ###
 ### USAGE:
+###   Synchronize the VMR to the commit of dotnet/installer that is currently being built:
 ###     ./vmr-sync.sh --tmp-dir "$HOME/repos/tmp"
+###
+###   Synchronize the VMR to a specific commit of dotnet/runtime using custom fork:
+###     ./vmr-sync.sh \
+###        --repository runtime:e7e71da303af8dc97df99b098f21f526398c3943 \
+###        --remote runtime:https://github.com/yourfork/runtime \
+###        --tmp-dir "$HOME/repos/tmp"\
+###
 ### Options:
 ###   -t, --tmp, --tmp-dir PATH
 ###       Required. Path to the temporary folder where repositories will be cloned
+###   --repository name:GIT_REF
+###       Optional. Repository + git ref separated by colon to synchronize to.
+###       This can be a specific commit, branch, tag..
+###       When omitted, the script will synchronize the installer commit based on the version of the parent
+###       where this script is stored.
 ###   -v, --vmr, --vmr-dir PATH
 ###       Optional. Path to the dotnet/dotnet repository. When null, gets cloned to the temporary folder
 ###   -b, --branch, --vmr-branch BRANCH_NAME
 ###       Optional. Branch of the 'dotnet/dotnet' repo to synchronize to
-###       This should match the target branch of the PR, defaults to 'main'
-###   --target-ref GIT_REF
-###       Optional. Git ref to synchronize to. This can be a specific commit, branch, tag..
-###       Defaults to the revision of the parent installer repo
-###   --debug
-###       Optional. Turns on the most verbose logging for the VMR tooling
+###       This should match the target branch of the PR; defaults to 'main'
+###   --remote name:URI
+###       Optional. Additional remote to use during the synchronization
+###       This can be used to synchronize to a commit from a fork of the repository
+###       Example: 'runtime:https://github.com/yourfork/runtime'
+###   --recursive
```

Comment:
This documentation should be updated to indicate the default value changes per the use of the `--repository` option.